apps/developer-portal/src/components/users/user-role-permissions.tsx

```diff
@@ -103,7 +110,7 @@ export const UserRolePermissions: FunctionComponent<UserRolePermissionsInterface
                         <Grid.Row column={ 1 }>
                             <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 8 }>
                                 <LinkButton
-                                    data-testid={ `${ props[ `data-testid` ] }_back_button` }
+                                    data-testid={ `${ testId }_back_button` }
```

Comment:
Shall we go with dashes instead of underscores?

Reply:
Fixed with f762024